root_data.tf

```diff
@@ -0,0 +1,7 @@
+data "archive_file" "log_data_lambda" {
+  type        = "zip"
+  source_file = "./tdr-terraform-modules/lambda/functions/log-data/lambda_function.py"
```

Comment:
This is referencing the `tdr-terraform-modules` should this actual be moved to the `da-terraform-modules`?

Reply:
This is specific lambda code for this lambda so I don't want to put it in `da-terraform-modules` It doesn't really belong in `tdr-terraform-modules` either so I've copied the python file into this project and referenced that.